packages/cozy-sharing/src/components/ShareModal/SharingDetailsModal.jsx

```diff
@@ -44,7 +44,7 @@ export const SharingDetailsModal = ({
               name={t(`${documentType}.share.sharedBy`, {
                 name: getDisplayName(owner)
               })}
-              url={owner.instance}
+              url={owner?.instance}
```

Comment:
if the owner didn't exist we should not display the OwnerIdentity component in the first place